app/templates/buyers/brief_publish_confirmation.html

```diff
@@ -35,9 +35,8 @@
         {% include "toolkit/page-heading.html" %}
     {% endwith %}
     <p class="padding-bottom-small">All requirements are published on the Digital Marketplace where anyone can see them.</p>
-
-    <p><strong>Suppliers will be able to apply for:</strong></p>
-    <p class="padding-bottom-small">2 weeks ending at midnight.</p>
+    <p class="padding-bottom-small">All requirements are open for {{ dates.application_open_weeks }} weeks.</p>
+    <p class="padding-bottom-small">If you publish your requirements today ({{ dates.today }}), suppliers will be able to apply until {{ dates.closing_time }}, {{ dates.closing_date }}.</p>
 
     <p><strong>Supplier questions will be sent to:</strong></p>
```

Comment:
Can we use the `.dmspeak` styles for this block? The `padding-bottom-small` classes came across from the prototype so we should remove them in app code when possible.

Also, I'd argue `Supplier questions will be sent to:` should be in the same paragraph as the email address so there's no break in the sentence between the two.